scriptmodules/helpers.sh

```diff
@@ -901,6 +904,22 @@ function loadModuleConfig() {
     done
 }
 
+## @fn getSystemExtensions()
+## @param system sytem name to fetch extensions for
+## @brief Prints a space separated string of disk extensions supported by the system 
+## @details Example: ".a26 .bin .rom .zip .gz"
+function getSystemExtensions() {
+    local config
+    for config in "$configdir/all/platforms.cfg" "$scriptdir/platforms.cfg"; do
+         [[ -f "$config" ]] && break
+    done
+    
+    iniConfig "=" '"' "$config"
+    iniGet "$1_exts"
+
+    echo $ini_value
```

Comment:
depends what you are printing of course - but in other cases it would matter

```
test="    hello world"
echo $test
```

in this case it won't matter - but it's good practise to avoid problems